cmd/database.go

```diff
@@ -26,16 +26,15 @@ import (
 
 var schemaOnly bool
 
-var listDatabasesCmd = &cobra.Command{
-	Use:     "databases",
-	Aliases: []string{"projects"},
-	Short:   "Lists databases/projects",
-	Long:    "Project is an alias for Database - this command will list all projects/databases",
+var listProjectsCmd = &cobra.Command{
+	Use:   "projects",
```

Comment:
Can we add 'database(s)' alias to the commands, so it's become backward compatible change?

Reply:
It was like this (projects) was an alias. We discussed and decided to make the cut over to project completely.

Reply:
Then it'll break few places where CLI is used.
CLI integration tests, YCSB, backups maybe.

Reply:
Yes it is breaking change.